### DISCUSSION TITLE
How can i stop hacompanion to flood the syslog

### DISCUSSION THREAD

**Discussion by Bram-diederik** [tobias-kuendig/hacompanion#42] [Q&A] [answered]
I had a crash and i cant see journalctl proper because hacompanion floods the logs.

How to make hadompanion less heavy on the logs. Its a raspberry pi

**Comment by pschmitt** [comment]
A log excerpt might be useful here, what's in there?

**Comment by ensonic** [comment]
Maybe we can add a flag (`--quiet`) and set that when run from systemd to only log issues.

Right now the log is quite chatty and that's maybe helpful when running it manually to initially configure it correctly.

**Reply by tobias-kuendig** [reply, marked as answer]
implemented in v1.0.20

**Reply by ensonic** [reply]
Thanks a lot!
